api/camunda/generation-strategy.js

```diff
@@ -17,32 +28,45 @@ module.exports = {
   postGenerateDocs,
 };
 
-function hackChangesetDescription() {
-  // This is a temporary hack, until https://github.com/camunda/camunda-docs/issues/3568 is resolved.
-  //   The OpenAPI generator plugin we're using does not use the correct `description` property
-  //   for the `UserTaskUpdateRequest` object. Instead of picking up the actual property description,
-  //   it picks up the description of the first merged schema in the `allOf` property (i.e. from the `Changeset` schema).
-  // This adjustment replaces the description of the `Changeset` schema with the current description of
-  //   the `UserTaskUpdateRequest.changeset` property.
-  console.log("hacking changeset description...");
-  replace.sync({
-    files: `${specFile}`,
-    from: /^      description: A map of changes.$/m,
-    to: `      description: |
-        JSON object with changed task attribute values.
-
-        The following attributes can be adjusted with this endpoint, additional attributes
-        will be ignored:
-
-        * \`candidateGroups\` - reset by providing an empty list
-        * \`candidateUsers\` - reset by providing an empty list
-        * \`dueDate\` - reset by providing an empty String
-        * \`followUpDate\` - reset by providing an empty String
-
-        Providing any of those attributes with a \`null\` value or omitting it preserves
-        the persisted attribute's value.
+function temporarilyAddCreateProcessInstanceExamples() {
+  // This is a temporary hack, to add example requests to the `Create Process Instance` endpoint.
+  //   After the upcoming release, this will be incorporated into the source spec, but the source spec is currently frozen.
+  return {
+    from: /\$ref: "#\/components\/schemas\/CreateProcessInstanceRequest"\n      responses:/m,
+    to: `$ref: "#/components/schemas/CreateProcessInstanceRequest"
+            examples:
+              "By process decision key":
+                summary: "Create a process instance by processDefinitionKey."
+                value:
+                  processDefinitionKey: 12345
+                  variables: {}
+              "By BPMN process ID":
+                summary: "Create a process instance by bpmnProcessID and version."
+                value:
+                  bpmnProcessId: "1234-5678"
+                  version: 1
+                  variables: {}
+      responses:`,
+  };
+}
```

Comment:
I'll backport the changes into the release branch. This shouldn't be necessary then, I guess?

Reply:
Done with https://github.com/camunda/camunda/pull/22715